test/modules/ModuleLib/TestResultReportingClient.cs

```diff
@@ -264,13 +278,20 @@ public partial class ApiException<TResult> : ApiException
     {
         public TResult Result { get; private set; }
 
-        public ApiException(string message, int statusCode, string response, System.Collections.Generic.IReadOnlyDictionary<string, System.Collections.Generic.IEnumerable<string>> headers, TResult result, System.Exception innerException) 
+        public ApiException(string message, int statusCode, string response, System.Collections.Generic.IReadOnlyDictionary<string, System.Collections.Generic.IEnumerable<string>> headers, TResult result, System.Exception innerException)
             : base(message, statusCode, response, headers, innerException)
         {
             Result = result;
         }
     }
 
+    class AnyErrorDetectionStrategy : ITransientErrorDetectionStrategy
+    {
+        public bool IsTransient(Exception ex)
+        {
+            return true;
+        }
+    }
```

Comment:
I see no harm on retrying on any error (don't want to miss one by manually specifying).

Reply:
We don't want this cuz if the message is missing... that can be an issue on edgeHub, right?

Reply:
I don't understand. This is retrying just the trc reporting.